packages/shared/lib/core/ledger/constants/use-ledger-simulator.constant.ts

```diff
@@ -1 +1 @@
-export const USE_LEDGER_SIMULATOR = false
+export const USE_LEDGER_SIMULATOR = true
```

Comment:
i think we want this disabled, right?

Reply:
Yes, did you disable for testing? @Tuditi